src/test/java/org/openmrs/performance/registries/DoctorRegistry.java

```diff
@@ -94,6 +94,14 @@ public ChainBuilder openLabResultsTab(String patientUuid) {
 	public ChainBuilder openAllergiesTab(String patientUuid) {
 		return exec(httpService.getAllergies(patientUuid));
 	}
+
+	public ChainBuilder addAllergies(String patientUuid) {
+		return exec(
+			httpService.searchPatient("test location test"),
```

Comment:
We don't need to search for the patient again as the user at this point is already in a patient's chart page (according to the visit patient scenario)